### ISSUE TITLE
Make filtering a single station easier

### ISSUE THREAD

**Issue by aroffringa** [lofar-astron/DP3#207]
The `filter.baseline` syntax is quite complex, and obviously I never understood it even I tried it a few times. It's already complicated because baselines need to be specified that are to be kept, while it's also common to remove a few stations because they didn't behave properly. It would be helpful to have a `filter.excludeantenna` or something like that, that can be given one or few station names that it removes.

**Comment by rvweeren** [comment]
I agree. For example, I always have to check the documentation to find the syntax for removing a station, it's too complicated for me to remember...
